src/assets/styles/app.css

```diff
@@ -1,103 +0,0 @@
-/*
- * Fonts
- */
-@import url(https://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,700italic,300,400,700);
-
-/*
- * Media
- */
-.media .img {
-  float: left;
-  margin-right: 10px;
-}
-
-/*
- * Meta
- */
```

Comment:
.meta is not in use and is removed from https://github.com/PRX/styleguide.prx.org/pull/20/files#diff-478c8efeb2d9a5f992f435f5dcd2afb5R1